src/app/services/admin-panel.guard.ts

```diff
@@ -12,31 +12,31 @@ export class AdminPanelGuard implements CanLoad, CanActivate {
   }
 
   /**
-   * Function that allow user to navigate to other route modules if it returns true
+   * Function that allows user to navigate to the current route module if it returns true
    * checks permission every time
-   * @returns true if user's role is not ROLE_ADMIN
-   * @returns false if user's role is ROLE_ADMIN
+   * @returns true if user's role is ROLE_ADMIN
+   * @returns false if user's role is not ROLE_ADMIN
    */
   canActivate(
     route: ActivatedRouteSnapshot,
     state: RouterStateSnapshot): boolean  {
     if (this.isAdmin()) {
```

Comment:
it could be simplified. return isAdmin()

Reply:
fixed